src/jquery.sceditor.js

```diff
@@ -69,8 +69,8 @@ define(function (require) {
 
 		options = options || {};
 
-		if (!options.runWithoutWysiwygSupport && !browser.isWysiwygSupported) {
-			return;
+		if (options.runWithoutWysiwygSupport || !browser.isWysiwygSupported) {
+			SCEditor.isWysiwygSupported = false;
```

Comment:
If `runWithoutWysiwygSupport` is set to `false` and `isWysiwygSupported` is set to `false` the editor will still run which is incorrect. It should only run without WYSIWYG support when `runWithoutWysiwygSupport` is `true`.